### PR TITLE
Add probes for Kibana

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature

kind flake

**What this PR does / why we need it**:
Add liveness and readiness probe for Kibana.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
Imho we should also add the liveness probe...

**Comment by ialidzhikov** [comment]
The liveness probe is added.

File: cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: fluentd-es-config-v0.2.0
   namespace: kube-system
   labels:
+    kubernetes.io/cluster-service: "true"
```
Comment (monotek) [inline cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml]: Why is this added? As far as a know this label is deprecated.

**Review by monotek — CHANGES_REQUESTED** [review]

File: cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml (hunk 1 above)
Comment (ialidzhikov) [reply cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml]: Yep, you are right. Removed it.

**Comment by ialidzhikov** [comment]
Rebased against the master branch. Hope this helps with the failing test.

**Review by monotek — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
@monotek can you also approve? :)

I guess...
/kind feature

**Review by monotek — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
@monotek This will require a `/approve` :stuck_out_tongue:

**Comment by monotek** [comment]
I've already approved via checkbox in the review message box 😁

Sure I need to also add /approve?

**Comment by nikhita** [comment]
> I've already approved via checkbox in the review message box

Some repos are configured with the review checkbox = approve option, but some are not (k/k is not one of them) :upside_down_face:

**Comment by monotek** [comment]
Thanks for the hint :)
